### DISCUSSION TITLE
How can I change the text of the notification message ?

### DISCUSSION THREAD

**Discussion by Snej-Corleone** [jef/streetmerchant#2909] [Help]
Hello everybody, I am relatively new to the street merchant script. I would like to change the message send with telegram, Twitter etc., but I can't find the folder for this. Can anyone help me ? Thanks !

**Comment by rasurbrand** [comment]
You will find the .ts file in src\messaging

**Reply by Snej-Corleone** [reply]
I found the Twitter and telegram .ts , but there is noch string oder text for my message.
